impl/src/test/groovy/com/okta/sdk/impl/resource/SessionsTest.groovy

```diff
@@ -0,0 +1,126 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.impl.resource
+
+import com.okta.sdk.authc.credentials.TokenClientCredentials
+import com.okta.sdk.cache.CacheManager
+import com.okta.sdk.client.Client
+import com.okta.sdk.impl.api.ClientCredentialsResolver
+import com.okta.sdk.impl.api.DefaultClientCredentialsResolver
+import com.okta.sdk.impl.cache.DisabledCacheManager
+import com.okta.sdk.impl.client.DefaultClient
+import com.okta.sdk.impl.config.ClientConfiguration
+import com.okta.sdk.impl.ds.InternalDataStore
+import com.okta.sdk.impl.ds.JacksonMapMarshaller
+import com.okta.sdk.impl.http.RequestExecutor
+import com.okta.sdk.impl.resource.session.DefaultCreateSessionRequest
+import com.okta.sdk.impl.resource.session.DefaultSession
+import com.okta.sdk.impl.resource.user.DefaultUser
+import com.okta.sdk.impl.util.BaseUrlResolver
+import com.okta.sdk.impl.util.DefaultBaseUrlResolver
+import com.okta.sdk.resource.session.Session
+import org.testng.annotations.Test
+
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.*
+
+import static org.mockito.Mockito.*
+
+/**
+ * Tests for the Sessions API.
+ *
+ * @since 0.10.0
+ */
+class SessionsTest {
+
+    @Test
+    void simpleCreateSessionTest() {
+
+        ClientConfiguration clientConfig = new ClientConfiguration()
+        Client client = new DefaultClient(
+                new DefaultClientCredentialsResolver(new TokenClientCredentials("foobar")),
+                new DefaultBaseUrlResolver("https://example.com"),
+                null,
+                new DisabledCacheManager(),
+                clientConfig.getAuthenticationScheme(),
+                clientConfig.getRequestAuthenticatorFactory(),
+                clientConfig.getConnectionTimeout()) {
+            protected InternalDataStore createDataStore(RequestExecutor requestExecutor,
+                                                        BaseUrlResolver baseUrlResolver,
+                                                        ClientCredentialsResolver clientCredentialsResolver,
+                                                        CacheManager cacheManager) {
+                JacksonMapMarshaller mapMarshaller = new JacksonMapMarshaller()
+                Map data = mapMarshaller.unmarshal(this.getClass().getResource( '/stubs/sessions.json' ).text)
+
+                final InternalDataStore dataStore = mock(InternalDataStore)
+                DefaultSession session = new DefaultSession(dataStore, data)
+                when(dataStore.create((String) eq("/api/v1/sessions"), any(), (Class) eq(Session.class))).thenReturn(session)
+                return dataStore
+            }
+        }
+
+        Session createdSession = client.createSession(new DefaultCreateSessionRequest(null, null)
+                                                                .setSessionToken("101W_juydrDRByB7fUdRyE2JQ"))
```

Comment:
Is it nicer to store the sessionToken in a variable like `mockSessionToken` and reference it in these places?